lib/data/clientTools.ts

```
@@ -0,0 +1,70 @@
+): Promise<Record<string, ToolRequestBody> | null> => {
+  if (customerEmail) {
+    const record = await db.query.cachedClientTools.findFirst({
+      where: eq(cachedClientTools.customerEmail, customerEmail),
+    });
+    if (record) return record.tools as Record<string, ToolRequestBody>;
+  }
+  const globalRecord = await db.query.cachedClientTools.findFirst({
+    where: and(
+      isNull(cachedClientTools.customerEmail),
+      isNull(cachedClientTools.platformCustomerId),
+    ),
```

ESLint

[error] Replace `⏎······isNull(cachedClientTools.customerEmail),⏎······isNull(cachedClientTools.platformCustomerId),⏎····` with `isNull(cachedClientTools.customerEmail),·isNull(cachedClientTools.platformCustomerId)`

(prettier/prettier)